lib/mblinks.js

```diff
@@ -100,7 +130,13 @@ var MBLinks = function (cachekey, expiration) {
         insert_func(mblinks.createMusicBrainzLink(mb_url, _type));
       });
     } else {
-      mblinks.ajax_requests.push($.proxy(function () {
+      // Merge with previous context if there's already a pending ajax request
+      var handlers = [];
+      if (url in mblinks.ajax_requests) {
+        handlers = mblinks.ajax_requests[url].context.handlers;
+      }
+      handlers.push(insert_func);
+      mblinks.ajax_requests.push(url, function () {
```

Comment:
I wonder if `url` is the correct thing to use here, i would use `query` defined as : 
```javascript
var query = mblinks.mb_server + '/ws/2/url?resource=' + encodeURIComponent(url) + '&inc=' + mb_type + '-rels';
```
We may do ajax requests for the same `url` but with different mb_type (it will be the case if we want to support MB places; which are labels on discogs side).